### ISSUE TITLE
Feature: Describe OSCR rating

### ISSUE THREAD

**Issue by zanieb** [open-sauced/app#3854]
### Suggested solution

e.g.  at
<img width="1900" alt="Screenshot 2024-08-05 at 12 19 17 PM" src="https://github.com/user-attachments/assets/3dbf9a23-263f-4cca-91fa-db7b5774eed5">

I see an OSCR rating but have no clue what the scale is or what it means. A hover tooltip with more information would be nice, as well as a link to more information.

**Comment by bdougie** [comment]
@zanieb you caught us between releases, we will have a blog post and other elements in the UI explaining the OSCR soon. 

cc @BekahHW to follow up here.

**Comment by BekahHW** [comment]
@zanieb here are the initial [docs](https://opensauced.pizza/docs/features/contributor-insights/#open-source-contributor-rating-oscr) for the OSCR. Feel free to ask questions or give feedback.

**Comment by zanieb** [comment]
Thanks for the prompt replies.

Some small things

- 30 days seems to short for a rolling metric — it seems weird that a score would drop significantly if someone went on vacation for a week.
- There is no mention of the "highest possible" score

**Comment by jpmcb** [comment]
> 30 days seems to short for a rolling metric

True! We actually recently updated this to be a rolling 90 day metric. Part of this limitation is compute: if we receive good feedback here, it's likely we'd expand this out to 180 days or even 1 year with more infrastructure behind it.

> There is no mention of the "highest possible" score

We also recently updated this: the previous highest score was 100 on a scale from 0-100. Anything above 80 is sublime. This score reflected 2 pillars: contributor quality and contributor "confidence". Quality is a measure of how frequently someone's contributos are accepted (i.e., a high score is contributions that are always accepted). Quality also reflects if someone has PRs frequently marked as spam (where their score will go to zero). Confidence is how often someone stars or forks something and comes back to make a contribution.

We updated this recently where the current highest possible score is 300 and reflects 3 pillars: quality, confidence, and "sway". This new pillar, sway, is a measure of a contributors ability to drive conversation within open source ecosystems. We look at comments, issues, pull requests, and reviews to measure this.

With the new top score, anything above 250 is extremely good. Above 200 is amazing. Above 100 indicates someone is making quality contributions and may be ready for mentorship or maintainership. Above 50 indicates strong, quality contributions. And above 0 is likely someone first starting out.
